charts/dvpe-deployment-gloo/templates/secret.yaml

```diff
@@ -2,58 +2,65 @@
 
 {{- if .Values.externalSecrets.service.key }}
 ---
-apiVersion: 'kubernetes-client.io/v1'
+apiVersion: 'external-secrets.io/v1beta1'
 kind: ExternalSecret
 metadata:
   name: {{ $serviceName }}-service-secrets
   namespace: {{ .Release.Namespace }}
 spec:
-  backendType: secretsManager
-  {{- if .Values.externalSecrets.service.roleArn }}
-  roleArn: {{ .Values.externalSecrets.service.roleArn }}
-  {{- end}}
+  secretStoreRef:
+    kind: SecretStore
+    name: {{ .Values.externalSecrets.service.secretstoreName | default (printf "external-secret-store-%s" .Release.Namespace) }}
```

Comment:
I don't think we need to make that configurable since the SecretStore is a generated resource and therefore the name would never change. If projects want to reference a custom SecretStore they need to define that on their own.